universum/modules/error_state.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+import inspect
+
+from ..lib.gravity import Module, Dependency
+
+
+class GlobalErrorState(Module):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)  # type: ignore
```

Comment:
codestyle question: should I use 'super()' in similar code, or 'super(GlobalErrorState, self)' ?

Reply:
Use super().
super(GlobalErrorState, self) is only needed to support python 2.